src/main/java/com/hillel/items_exchange/controller/UserController.java

```diff
@@ -43,4 +52,52 @@
                         Locale.getDefault()))
                 ));
     }
+
+    @GetMapping("/child")
+    public ResponseEntity<List<ChildDto>> getChildren(Principal principal) {
+        return new ResponseEntity<>(userService.getChildren(getUser(principal.getName())), HttpStatus.OK);
+    }
+
```

Comment:
And what about the case when User doesn`t have children?

Reply:
It's returning empty ArrayList now. Is this is ok?